components/sections/AboutUs.jsx

```diff
@@ -0,0 +1,48 @@
+import Image from "next/image";
+import { useState } from "react";
+import { data } from "/utils/aboutUsData";
+
+export default function AboutUs() {
+    const [selectedDiv, setSelectedDiv] = useState(null);
+    function changeState(id) {
+        if (selectedDiv === id) return setSelectedDiv(null);
+        setSelectedDiv(id);
+    }
```

Comment:
Can we name this function a more descriptive name related to what it does so that it would be easier to understand but if you can't find a better name this one is okay.

Reply:
will brainstorm more.